customerClient/src/main/java/org/lilachshop/customerclient/App.java

```diff
@@ -29,7 +29,7 @@ public void init() throws Exception {
 
     @Override
     public void start(Stage stage) throws IOException {
-        scene = new Scene(loadFXML("main"));
+        scene = new Scene(loadFXML("MessageBoxComplaint"));
```

Comment:
Please do not push flow dependent changes, in this case this this was probably not wanted in master.